resources/js/app.js

```diff
@@ -1,8 +1,12 @@
 import Vue from 'vue'
 
-// Register all the Vue components
-const files = require.context('./', true, /\.vue$/i)
-files.keys().map(key => Vue.component(key.split('/').pop().split('.')[0], files(key).default))
+// Register global Vue components
+Vue.component('Layout', require('./Misc/Layout').default);
```

Comment:
If you first register the components you want globally, it will keep them in the main chunk (app.js)